docs/sanitizers/asan-flags.md

```diff
@@ -0,0 +1,132 @@
+---
+title: "AddressSanitizer Runtime Options"
+description: "List of runtime options for the Microsoft C/C++ Address Sanitizer"
+ms.date: 8/05/2025
+helpviewer_keywords: ["AddressSanitizer options", "AddressSanitizer flags", "Address Sanitizer options", "Address Sanitizer flags", "asan options", "asan flags", "sanitizer flags", "asan_options", "AddressSanitizerFlags", "CommonSanitizerFlags"]
+---
+
+# AddressSanitizer Runtime Options
+
+The following is a list of supported runtime options that can be set by the `ASAN_OPTIONS` environment variable or by overriding the `__asan_default_options()` user function. Reference: [AddressSanitizer Runtime Options](./asan-runtime.md#runtime-options). Note that while these mirror the options available in Clang's ASan, the description will differ by varying amounts in consideration of the MSVC ASan runtime.
+
+This is a living document that may update at any time due to the constant development of AddressSanitizer on the Windows platform. If you discover options that aren't accurately documented, we invite you to [report a bug](https://aka.ms/feedback/report?space=62).
+
+| Flag | Default value | Description |
+|:-----|:--------------|:------------|
+|abort_on_error | false | If true, the tool calls `abort()` instead of `_exit()` after printing the error report.|
+|alloc_dealloc_mismatch | false | Enables runtime detection of mismatched memory operations such as `malloc`/`delete`, `new[]`/`free`, etc.|
+|allocator_frees_and_returns_null_on_realloc_zero | 1 | 1 - realloc(p, 0) is equivalent to free(p). 0 - realloc(p, 0) will return a non-`NULL` value.|
+|allocator_may_return_null | false | If true, the allocator will return `nullptr` and fail when out of memory instead of crashing.|
+|allocator_release_to_os_interval_ms | -1 | Experimental. 64-bit allocator only. Specify minimum time interval in ms between attempts to release memory |
+|allow_addr2line | false | If true, the tool uses the `addr2line` binary to symbolize stack traces when `llvm-symbolizer` is not available.|
+|allow_user_poisoning | true | If true, user may manually mark memory regions as poisoned or unpoisoned.|
+|atexit | false | If true, prints ASan exit stats even after program terminates successfully.|
+|can_use_proc_maps_statm | true | If false, do not attempt to read `/proc/maps/statm`. Mostly useful for testing sanitizers. (Not implemented - `/proc/maps/statm` doesn't exist on Windows)|
+|check_initialization_order | false | If true, attempts to catch initialization order issues.|
+|check_malloc_usable_size | true | Allows the users to work around the bug in Nvidia drivers prior to 295.*.|
```

Comment:
Please reconsider all places where you use 'user' in the doc. Probably could be replaced with 'you', yes?